pkg/config/config.go

```diff
@@ -543,7 +509,9 @@ func (f *File) Write(path string) (err error) {
 	if err != nil {
 		return fmt.Errorf("error creating config file: %w", err)
 	}
-	if err := toml.NewEncoder(fp).Encode(f); err != nil {
+	encoder := toml.NewEncoder(fp)
+	encoder.Indentation("")
```

Comment:
What problem is `.Indentation` solving? 

I'd like us to add a `// NOTE: ...` to clarify why we use it.

Reply:
UPDATE: I think I see why it's used in the `config_test.go` below.

Reply:
Sure, I will add a comment.

Reply:
Done.

Reply:
To clarify the original question, this is removing the leading spaces. 
I'm not sure it qualifies as a problem here but I don't think it's needed either.